src/styles/components/_tabs.scss

```diff
@@ -37,5 +37,5 @@ nav.tabs {
 }
 
 nav.tabs + * {
-  --flow-spacing: var(--gcds-spacing-400);
+  margin-block-start: var(--gcds-spacing-400);
```

Comment:
Same comment here.

Reply:
A bit more details. This will add margin on everything, whereas flow-spacing will add margin in between everything.

Reply:
True, I missed that it didn't just add the `margin-block` start to the element itself.
I will rewrite it to add the utility classes directly to the table partial instead of applying them to the CSS.